test/TesterAzureUtils/SiloInstanceTableManagerTests.cs

```diff
@@ -22,34 +24,29 @@ public class SiloInstanceTableManagerTests : IClassFixture<SiloInstanceTableMana
     {
         public class Fixture
         {
-            public Fixture()
-            {
-                LogManager.Initialize(new NodeConfiguration());
-            }
+            public ILoggerFactory LoggerFactory { get; set; } =
+                TestingUtils.CreateDefaultLoggerFactory(new NodeConfiguration().TraceFileName);
```

Comment:
If this is created by the Fixture class, the Fixture class should also implement IDisposable and dispose the factory when the fixture is being disposed